plugins/scripting/scripting-compiler-impl/src/org/jetbrains/kotlin/scripting/definitions/KotlinScriptDefinitionProvider.kt

```diff
@@ -26,6 +26,9 @@ interface ScriptDefinitionProvider {
     companion object {
         fun getInstance(project: Project): ScriptDefinitionProvider? =
             ServiceManager.getService(project, ScriptDefinitionProvider::class.java)
```

Comment:
In Kotin you can use `project.service<ScriptDefinitionProvider>()`.

Reply:
I cannot use it here because it is compiler module, there is no platform-impl on the classpath (and in deepdencies)